src/components/depoimentos-section/testemonials-carousel/index.tsx

```diff
@@ -0,0 +1,125 @@
+    setDirection("right");
+    if (activeProjectIndex === mock_depoimentos_datas.length - 1) {
+      return setActiveProjectIndex(0);
+    }
+
+    return setActiveProjectIndex((prev) => prev + 1);
+  };
+  const motionInitial = direction === "right" ? "hiddenRight" : "hiddenLeft";
+  const slideVariants = {
+    hiddenRight: {
+      x: "100%",
+      opacity: 0,
+    },
+    hiddenLeft: {
+      x: "100%",
+      opacity: 0,
+    },
+    visible: {
+      x: "0",
+      opacity: 1,
+      transition: {
+        duration: 0.8,
+      },
+    },
+    exit: {
+      opacity: 0,
+      x: direction === "right" ? "-100%" : "100%",
+      transition: {
+        duration: 0.4,
+      },
+    },
+  };
+
+  const Pagination = () => {
+    return mock_depoimentos_datas.map((_, index) => (
+      <button
+        key={index}
+        className={`${styles.dot} ${index === activeProjectIndex ? styles.activeDot : ""}`}
+        onClick={() => setActiveProjectIndex(index)}
+        aria-label={`Ir para o slide ${index + 1}`}
+      />
+    ));
+  };
+
+  return (
+    <>
+      <div
+        role="region"
+        aria-roledescription="carousel"
+        aria-label="Carrossel de depoimentos da Abre"
+        className={styles.carousel}
+      >
+        <div className={styles.carousel_images}>
+          <MotionConfig reducedMotion="user">
+            <AnimatePresence mode="wait">
+              <motion.ul
+                key={activeProjectIndex}
+                initial={motionInitial}
+                animate="visible"
+                exit="exit"
+                variants={slideVariants}
+                className={styles.carousel_images_container}
+              >
+                <Slide
+                  aria-roledescription={`Slide ${activeProjectIndex + 1} de ${
+                    mock_depoimentos_datas.length
+                  } de depoimentos da abre`}
+                  quote={mock_depoimentos_datas[activeProjectIndex].quote}
+                  content={mock_depoimentos_datas[activeProjectIndex].description}
+                  author={mock_depoimentos_datas[activeProjectIndex].author}
+                />
+              </motion.ul>
+            </AnimatePresence>
+          </MotionConfig>
+        </div>
+        <div
+          aria-label="Controles do Carrossel"
+          className={styles.carousel_nav}
+        >
+          <button
+            aria-label="Slide anterior"
+            className={styles.left}
+            onClick={handlePreviousSlide}
+          >
+            <FaChevronLeft
+              aria-hidden="true"
+              size={26}
+            />
+          </button>
+          <button
+            aria-label="Próximo slide"
+            className={styles.right}
+            onClick={handleNextSlide}
+          >
+            <FaChevronRight
+              aria-hidden="true"
+              size={26}
+            />
+          </button>
+          <div className={styles.pagination}>{Pagination()}</div>
```

Comment:
Se esse Pagination for algo que a gente vai utilizar em outros lugares acho que vale separar em outro componente, senão acho que pode renderizar ele direto aqui, senão cada vez que mudar de slide essa função vai ser recriada

Reply:
Sim, eu esqueci de tirar

Reply:
Essa paginação é somente aqui.

Reply:
Não achei necessário fazer a componetização pois é somente usado aqui

Reply:
boa, então como o código é curto, acho que pode colocar direto no html, sem separar numa função

Reply:
Okay :)